packages/now-static-build/index.js

```diff
@@ -31,13 +39,70 @@ exports.build = async ({
     (config && config.distDir) || 'dist',
   );
 
-  if (path.basename(entrypoint) === 'package.json') {
+  const entrypointName = path.basename(entrypoint);
+  if (entrypointName === 'package.json') {
     await runNpmInstall(entrypointFsDirname, ['--prefer-offline']);
-    if (await runPackageJsonScript(entrypointFsDirname, 'now-build')) {
-      validateDistDir(distPath);
-      return glob('**', distPath, mountpoint);
+
+    const pkgPath = path.join(workPath, entrypoint);
+    const pkg = JSON.parse(readFileSync(pkgPath, 'utf8'));
+    console.log({ pkg });
+
+    let output = {};
+    const routes = [];
+    if (meta.isDev && pkg.scripts && pkg.scripts['now-dev']) {
+      if (nowDevScriptPromises.has(entrypoint)) {
+        console.log('`now-dev` server already running for %j', entrypoint);
+      } else {
+        // Run the `now-dev` script out-of-bounds, since it is assumed that
+        // it will launch a dev server that never "completes"
+        const devPort = await getPort();
+        console.log({ devPort });
+        const opts = {
+          env: { PORT: String(devPort) },
+        };
+        const promise = runPackageJsonScript(
+          entrypointFsDirname,
+          'now-dev',
+          opts,
+        );
+        promise.then(
+          () => {
+            nowDevScriptPromises.delete(entrypoint);
+          },
+          (err) => {
+            console.log('`now-dev` script error:', err);
+            nowDevScriptPromises.delete(entrypoint);
+          },
+        );
+        nowDevScriptPromises.set(entrypoint, promise);
+
+        // Now wait for the server to have listened on `$PORT`, after which we
+        // will ProxyPass any requests to that development server that come in
+        // for this builder.
+        await timeout(waitForPort('localhost', devPort), 60 * 1000);
+        console.log('Detected dev server for $j', entrypoint);
+
+        const srcBase = `/${mountpoint.replace(/^\.\//, '')}`;
```

Comment:
```suggestion
        const srcBase = mountpoint.replace(/^\.\//, '');
```

I think you are adding `/` twice